BankingKataSpecs/AccountSpecs/Features/CreateAccount.feature

```diff
@@ -0,0 +1,19 @@
+﻿Feature: Create account
+	In order to us my account
+	As a foo user
+	I want to be able to create an account.
+
+@mytag
+Scenario: New account has zero cash.
+	Given I want to create an account
```

Comment:
This `Given` seems redundant - can it be missed out? It's equivalent to a test which doesn't require any arrangement.